modules/navier_stokes/src/functormaterials/INSFVkEpsilonViscosityFunctorMaterial.C

```diff
@@ -41,7 +41,9 @@ INSFVkEpsilonViscosityFunctorMaterial::INSFVkEpsilonViscosityFunctorMaterial(
   addFunctorProperty<ADReal>(NS::mu_t,
                              [this](const auto & r, const auto & t) -> ADReal
                              {
-                               return _C_mu(r, t) * _rho(r, t) * Utility::pow<2>(_k(r, t)) /
-                                      std::max(NS::mu_t_low_limit, _epsilon(r, t));
+                               return std::max(NS::mu_t_low_limit,
```

Comment:
@tanoret , do you think this change makes sense? To always include a little viscosity?

Reply:
I think that was the initial intent here, why else why that variable named like that?